lib/path.js

```diff
@@ -25,10 +25,16 @@ const argsCountPerCommand = {
   a: 7,
 };
 
+/**
+ * @param {string} c
+ */
 const isCommand = (c) => {
```

Comment:
TS cannot infer all this, right?

Reply:
It cannot infer the argument types for standalone functions, but it can infer the return types.

However if you pass a closure to a typed function, then the arguments are inferred. For example if `data` is of type `string[]`, then `data.map(s => s.toUpperCase())`, it will infer that `s` is of type `string`.